client/db.go

```diff
@@ -208,6 +209,13 @@ type Store interface {
 
 	// ExecRequest executes the given GQL request against the [Store].
 	ExecRequest(context.Context, string) *RequestResult
+
+	// ACPModule returns the underlying acp module (optional). Incase the access control
+	// is turned off, then the returned acp module will have no value (will be empty).
+	ACPModule(context.Context) (immutable.Option[acp.ACPModule], error)
```

Comment:
todo: `client.Store` is transactional, and all members on it must respect the transaction.  These two new members do not respect the transaction, so please move them onto `client.DB`

Reply:
Sorry somewhat new to this side of the code, can you give me a quick intro the the difference between  Store and DB (other than Txn-ality)

- [ ] Move to client.DB

Reply:
txn is basically the only important aspect to remember IMO, anything else is likely subjective.

Reply:
Thanks for the explaination. As mentioned will do.

Reply:
This was fixed in the commit: ``` PR(ACP): Move `AddPolicy` from Store to DB ```

- [x] Move `AddPolicy` from `Store` to `DB`